widget/class-widget.php

```diff
@@ -111,6 +126,8 @@ public function widget( $args, $instance ) {
 
          </ul>
 
+         <p><a href="<?php echo $allOppsPageLink; ?>">View All</a></p>
```

Comment:
We need to check if there is a page first before we add this. It's possible an admin would create volunteer opportunities before adding one of the shortcodes to a page.